### PR TITLE
Basic backend with getArticles

### PR DESCRIPTION
**Pull request by waterwawe**
- Backend "skeleton" with one endpoint for getting articles
- Migrations added
- Identity added (but no auth and db tables yet)

### CODE REVIEW

File: src/Backend/FakeNews/FakeNews.Api/Controllers/ArticleController.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace FakeNews.Api.Controllers
+{
+    [Route("api/[controller]/[action]")]
+    [ApiController]
+    public class ArticleController : ControllerBase
```
Comment (TomSoldier) [inline src/Backend/FakeNews/FakeNews.Api/Controllers/ArticleController.cs]: Controller names use a plural by convention. It should be ArticlesController.

File: src/Backend/FakeNews/FakeNews.Api/Controllers/ArticleController.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace FakeNews.Api.Controllers
+{
+    [Route("api/[controller]/[action]")]
```
Comment (TomSoldier) [inline src/Backend/FakeNews/FakeNews.Api/Controllers/ArticleController.cs]: Use an explicit path. Even if the controller class name changes in the future, the path to the api endpoint will not change.

File: src/Backend/FakeNews/FakeNews.Dal/Entites/Article.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+namespace FakeNews.Dal.Entites
+{
+    public class Article
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public int CreatedByUserId { get; set; }
+        public string Content { get; set; }
+    }
```
Comment (TomSoldier) [inline src/Backend/FakeNews/FakeNews.Dal/Entites/Article.cs:15]: Additional fields are required to **categorize** news and set the **validity period** for news.

**Review by TomSoldier — CHANGES_REQUESTED** [review]

File: src/Backend/FakeNews/FakeNews.Api/Controllers/ArticleController.cs (hunk 1 above)
Comment (waterwawe) [reply src/Backend/FakeNews/FakeNews.Api/Controllers/ArticleController.cs]: Done

File: src/Backend/FakeNews/FakeNews.Dal/Entites/Article.cs (hunk 3 above)
Comment (waterwawe) [reply src/Backend/FakeNews/FakeNews.Dal/Entites/Article.cs:15]: They will be added with the features and migration scripts will be generated.

File: src/Backend/FakeNews/FakeNews.Api/Controllers/ArticleController.cs (hunk 2 above)
Comment (waterwawe) [reply src/Backend/FakeNews/FakeNews.Api/Controllers/ArticleController.cs]: Done

File: src/Backend/FakeNews/FakeNews.Dal/Entites/Article.cs (hunk 3 above)
Comment (TomSoldier) [reply src/Backend/FakeNews/FakeNews.Dal/Entites/Article.cs:15]: OK.

**Review by TomSoldier — APPROVED** [review]

**Pull request merged**